charts/loki-distributed/templates/role.yaml

```
@@ -0,0 +1,18 @@
+{{ if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "loki.fullname" . }}
+  labels:
+    {{- include "loki.labels" . | nindent 4 }}
+rules:
+- apiGroups:      
+  - extensions
+  resources:      
+  - podsecuritypolicies
+  verbs:          
+  - use
+  resourceNames:  
+  - {{ include "loki.fullname" . }}
```

Lists should be indented. API group should be `policy`.
```suggestion
  - apiGroups:      
      - policy
    resources:      
      - podsecuritypolicies
    verbs:          
      - use
    resourceNames:  
      - {{ include "loki.fullname" . }}
```